src/app/clusters/im-compatibility/compatibility-common.cpp

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
could we find a more focused name than 'common'?

common/util/support/helpers are all names that are vague and are not clear when reading code why one would include 'common.h' or 'support.h'. 

Event the description of `Contains the common functions for compatibilty` does not seem to tell me what this is.

Reply:
Renamed it to ember-compatibility-functions to indicate it is used for maintain ember internal states.